app/controllers/application_controller.rb

```diff
@@ -123,6 +124,7 @@ class ApplicationController < ActionController::Base
   # Disable all filters except set_locale.
   # (Used to streamline API and Ajax controllers.)
   def self.disable_filters
+    skip_before_action(:redirect_anonymous_users)
```

Comment:
@nimmolo:
Because graphql calls `disable_filters`, this disables `redirect_anonymous_users` for graphql.
Is that the right thing to do?

Reply:
Yes - I believe so. All users are anonymous for graphql **according to the way Rails identifies users**, the browser session cookie. It took me a long time to understand why, and the difference between these types of authentication, and I would love to spare you that trouble if i can... I'll send it in an email.

Reply:
I agree.  At least for the moment, I don't see any reason to have any filters on graphql.  We might need to consider limiting traffic on graphql later, but that's far away.